### PR TITLE
Update Avast.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/Avast.com.xml [hunk 1]
```diff
@@ -1,44 +1,49 @@
...
-		- foundation ¹
-		- \w\w.support ¹
-		- ts ²
 
-	¹ Redirects to http
-	² 403
+	Invalid certificate:
+		- ^	( should be redirected to www. )
+		- ans
+		- ts
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Avast.com.xml]: *.u.avast.com

File: src/chrome/content/rules/Avast.com.xml [hunk 2]
```diff
@@ -1,44 +1,49 @@
...
+		- foundation
 
-		- foundation ¹
-		- \w\w.support ¹
-		- ts ²
 
-	¹ Redirects to http
-	² 403
+	Invalid certificate:
+		- ^	( should be redirected to www. )
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Avast.com.xml]: [SSLLABS](https://www.ssllabs.com/ssltest/analyze.html?d=avast.com&s=208.43.71.133&latest) shows ok, `avast.com` is in the SAN.

Add `an.avast.com`

File: src/chrome/content/rules/Avast.com.xml [hunk 3]
```diff
@@ -1,44 +1,49 @@
...
 				-->
 	<target host="avast.com" />
-	<target host="ans.avast.com" />
+	<rule from="^https?://avast\.com/"
+		to="https://www.avast.com/" />
+		<test url="https://avast.com/" />
+
+	<target host="an.avast.com" />
+	<rule from="^http://an\.avast\.com/"
+		to="https://avast-co-jp-ldc.d3.sc.omtrdc.net/" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Avast.com.xml]: `ans.` seems to be the same with `an.`?

File: src/chrome/content/rules/Avast.com.xml [hunk 4]
```diff
@@ -48,28 +53,12 @@
...
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^\.avast\.com$" name="^(?:avastComGiCountry|s_vi)$" /-->
-	<!--securecookie host="^id\.avast\.com$" name="^PLAY_SESSION$" /-->
-	<!--securecookie host="^press\.avast\.com$" name="^x-avast-(?:pro|vbd)$" /-->
-	<!--securecookie host="^www\.avast\.com$" name="^avastComLocale$" /-->
-
-	<securecookie host="^\." name="^s_vi$" />
-	<securecookie host="^\w" name="." />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Avast.com.xml]: Are there any special reasons to remove `securecookie`?

File: src/chrome/content/rules/Avast.com.xml (hunk 2 above)
Comment (ivysrono) [inline src/chrome/content/rules/Avast.com.xml]: an.avast.com 使用了无效的安全证书。 该证书仅对下列域名有效： *.d3.sc.omtrdc.net, d3.sc.omtrdc.net 错误代码: SSL_ERROR_BAD_CERT_DOMAIN

File: src/chrome/content/rules/Avast.com.xml (hunk 4 above)
Comment (ivysrono) [inline src/chrome/content/rules/Avast.com.xml]: 删顺手了

File: src/chrome/content/rules/Avast.com.xml (hunk 3 above)
Comment (gloomy-ghost) [inline src/chrome/content/rules/Avast.com.xml]: /robots.txt

这个的确比较激进，不加也没问题……

**Pull request merged**